static/src/stylesheets/module/_weather.scss

```diff
@@ -104,7 +104,7 @@ $weather-large-size: 60px;
     }
 }
 
-.i.weather__icon {
+.weather__icon svg {
```

Comment:
It's mainly because svg is setting w/h of the icons to 32px as default and we want to change the size. I can use '.weather__icon svg' just to change the size of the SVG and leave everything else to just '.weather__icon' but it will just add another rule so I decided to do it all directly on svg.

Do we actually need that 'span' wrapping the svg? Can we add classes right to the SVG. I suppose it must be at more places where we actually want to change the size of the SVGs.